doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-dmn-based-proc.adoc

```diff
@@ -1,5 +1,5 @@
 [id='test-scenarios-background-dmn-based-proc']
-= Adding a background data in DMN based test scenarios
+= Adding a background data in DMN-based test scenarios
```

Comment:
Suggestion - rephrase references to "a background data":

Examples:
s/Adding a background data in/Adding background data objects to/Adding background data in
s/set a background data in/set background data objects in/set background data in